spark/src/main/scala/ai/chronon/spark/Driver.scala

```diff
@@ -732,8 +732,14 @@ object Driver {
     }
 
     def run(args: Args): Unit = {
-      val putRequest = args.metaDataStore.putConf(args.confPath())
-      val res = Await.result(putRequest, 1.hour)
+      val acceptedEndPoints = List("ZIPLINE_METADATA")
```

Comment:
Let's not use the old term `zipline` as much as we could. 
Could you make those configurable or read from a config file or read from command line params? 
```suggestion
      val acceptedEndPoints = List("CHRONON_METADATA")
```